src/site/stages/build/webpack/manifest-helpers.js

```diff
@@ -9,13 +9,13 @@ function getAppManifests(root) {
       // eslint-disable-next-line import/no-dynamic-require
       const manifest = require(file);
 
-      manifest.filePath = file;
-      manifest.entryFile = path.resolve(
-        root,
-        path.join(path.dirname(file), manifest.entryFile),
-      );
-
-      return manifest;
+      return Object.assign({}, manifest, {
```

Comment:
We were mutating an object that Node put in its require cache on each function all before, which led to problems if you ran this function more than once.